backend/app/views.py

```diff
@@ -30,6 +33,11 @@ def bundle():
     return send_from_directory("dist", "dist.bundle.js")
 
 
+@app.route("/main.css")
```

Comment:
Why is this needed?

Reply:
Well it wont serve the styles otherwise. We have no other file server running on the container than flask. That is not the normal way of doing things so it seems odd, usually static content is served with nginx or something through /var/www. :+1:

Reply:
Got it :+1: